### ISSUE TITLE
Build problems with .NET Core SDK 2.1.401

### ISSUE THREAD

**Issue by BendicoE** [fluentribbon/Fluent.Ribbon#628]
I have cloned the latest source and I'm trying to build everything in Visual Studio 2017, very recently updated. The build fails on the Fluent.Ribbon project with the following message:

C:\Program Files\dotnet\sdk\2.1.401\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.targets(41,3): error MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft..targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

I read somewhere in a forum that someone have had problems earlier with paket and the .NET Core SDK. Could I be facing a similar issue?

**Comment by batzen** [comment]
Do you have the full .net core sdk installed? I also had to install it by hand for some reason on some machines.

**Comment by BendicoE** [comment]
I just downloaded and installed 2.1.401 by hand, but unfortunately, it did not make difference. However, there is an other interesting message appearing int the build output:

paket.dependencies and paket.lock are out of sync in C:\Projects\Fluent.Ribbon-develop\Fluent.Ribbon-develop.
1>Please run 'paket install' or 'paket update' to recompute the paket.lock file.

I don't know anything about paket, but could this message be a clue?

**Comment by batzen** [comment]
That's an internal issue of paket. I also get these messages.
Are you using the latest version of visual studio? Like 15.8.2?

**Comment by BendicoE** [comment]
I found the problem:smiley:

My Visual Studio installation was missing the F# components. I thought I didn't need them, because I never use F# and it was not selected by default when I installed Visual Studio. However, a comment about F# in the Sdk.targets file (actually just above line 41 which caused the compiler error) led me to the conclusion that it might be needed anyway.

Running the Showcase app now. Looks great, packed with features!
